.cruft.json

```diff
@@ -7,7 +7,7 @@
       "name": "APPUiO Control API",
       "slug": "control-api",
       "parameter_key": "control_api",
-      "test_cases": "defaults insecure",
+      "test_cases": "defaults insecure withcronjob",
```

Comment:
Some boilerplate updates (GitHub actions) are missing, did you use `commodore component update -t withcronjob` to generate the new test case?

Reply:
I didn't. I didn't know I could do that, haha.